hooks/shapediver/parameters/useParameterImportExport.ts

```diff
@@ -0,0 +1,162 @@
+import {NotificationContext} from "@AppBuilderShared/context/NotificationContext";
+import {useShapeDiverStoreParameters} from "@AppBuilderShared/store/useShapeDiverStoreParameters";
+import {useCallback, useContext} from "react";
+import {useShallow} from "zustand/react/shallow";
+
+/**
+ * Hook for managing parameter import/export and reset functionality.
+ */
+export function useParameterImportExport(namespace: string) {
+	const {batchParameterValueUpdate} = useShapeDiverStoreParameters(
+		useShallow((state) => ({
+			batchParameterValueUpdate: state.batchParameterValueUpdate,
+		})),
+	);
+
+	const notifications = useContext(NotificationContext);
+
+	/**
+	 * Export parameters as JSON file
+	 */
+	const exportParameters = useCallback(async () => {
+		try {
+			// Get current parameter values from store
+			const parameters = useShapeDiverStoreParameters
+				.getState()
+				.getParameters(namespace);
+			const parameterValues: {[key: string]: any} = {};
+
+			Object.values(parameters).forEach((paramStore) => {
+				const param = paramStore.getState();
+				parameterValues[param.definition.id] = param.state.uiValue;
+			});
+
+			// Create JSON blob and download
+			const jsonContent = JSON.stringify(parameterValues, null, 2);
+			const blob = new Blob([jsonContent], {type: "application/json"});
+			const url = URL.createObjectURL(blob);
+
+			// Create download link
+			const link = document.createElement("a");
+			link.href = url;
+			link.download = `parameters_${namespace}_${new Date().toISOString().split("T")[0]}.json`;
+			document.body.appendChild(link);
+			link.click();
+			document.body.removeChild(link);
+			URL.revokeObjectURL(url);
+
+			notifications.success({
+				message: "Parameters exported successfully",
+			});
+		} catch (error) {
+			console.error("Error exporting parameters:", error);
+			notifications.error({
+				message: "Failed to export parameters",
+			});
+		}
```

Comment:
@LemToUp instead of console.error, let's report to Sentry

Reply:
1c02dbe28bf8225b435d8a59520c4883170edfe8